src/_data/LiveData.js

```diff
@@ -133,6 +134,6 @@ export const connect = async store => {
         api.events.on(key, data => store.dispatch(action(data)));
         api.events.on(key, () => window.console.warn);
     });
-
+    api.getResidences();
```

Comment:
@qingweibinary that call either belong there , or veryfEmailCard to my understanding. "See where we place such calls" like ?